### PR TITLE
Re-introducing colourised output with relevant flags

### PR DESCRIPTION
**Pull request by CGoodwin90**
Addresses https://github.com/uselagoon/lagoon-sync/issues/70

Colourised output re-introduced via https://github.com/Mandala/go-log

Output is colourised when in interactive mode & disabled when either `noCliInteraction` or `noColour` is flagged.

### CODE REVIEW

File: cmd/sync.go [hunk 1]
```diff
@@ -101,11 +103,16 @@ func syncCommandRun(cmd *cobra.Command, args []string) {
 			ProjectName,
 			SyncerType,
 			sourceEnvironmentName, targetEnvironmentName))
+		utils.SetColour(true)
```
Comment (bomoko) [inline cmd/sync.go:106]: I think that we can probably get rid of this and the `if` below and just simply pass in the value of noColor - we know it's always going to be set, so no need to worry about if'ing

File: cmd/sync.go [hunk 2]
```diff
@@ -182,6 +189,7 @@ func init() {
 	syncCmd.PersistentFlags().BoolVar(&SSHVerbose, "verbose", false, "Run ssh commands in verbose (useful for debugging)")
 	syncCmd.PersistentFlags().BoolVar(&noCliInteraction, "no-interaction", false, "Disallow interaction")
 	syncCmd.PersistentFlags().BoolVar(&dryRun, "dry-run", false, "Don't run the commands, just preview what will be run")
+	syncCmd.PersistentFlags().BoolVar(&noColour, "no-colour", false, "Output is not colourised")
```
Comment (bomoko) [inline cmd/sync.go:192]: I'm also wondering if we shouldn't just change this up to bring it in line with the `setColour` function - maybe we change this is be `--in-color`, with the default of `true`?

**Review by bomoko — CHANGES_REQUESTED** [review]

File: cmd/sync.go (hunk 2 above)
Comment (bomoko) [reply cmd/sync.go:192]: even "set-colour" wouldn't be a bad name for the arg?

Comment (CGoodwin90) [reply cmd/sync.go]: As discussed this morning updated to have colourisation removed only for non-interactive mode.

File: cmd/sync.go (hunk 1 above)
Comment (CGoodwin90) [reply cmd/sync.go:105]: Updated to have colourisation removed only for non-interactive mode.

**Pull request merged**